### PR TITLE
feat: expose protobuf

### PR DESCRIPTION
**Pull request by dignifiedquire**
This allows other modules to reuse the protobuf definition

cc @yusefnapora

### CODE REVIEW

**Comment by yusefnapora** [comment]
Awesome!

**Comment by dignifiedquire** [comment]
@diasdavid can I get some quick review?

File: src/index.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ const protobuf = require('protocol-buffers')
 const pbm = protobuf(require('./crypto.proto'))
 const c = require('./crypto')
 
+exports.protobuf = pbm
```
Comment (diasdavid) [inline src/index.js]: add to the docs

Comment (dignifiedquire) [inline src/index.js]: docs are a different PR, not yet merged

Comment (diasdavid) [inline src/index.js]: add to that PR then :)

**Pull request merged**